.travis.yml

```diff
@@ -65,6 +65,11 @@ jobs:
         - DESC="unit tests oraclejdk11"
         - CMD="mvn install"
 
+    - jdk: openjdk-ea
+      env:
+        - DESC="unit tests openjdk-ea"
+        - CMD="mvn install"
+
     - jdk: oraclejdk8
       if: type = push AND repo = eclipse/eclipse-collections
```

Comment:
can you please remove this `if` statement so that the 12-EA build runs for all PRs and merges? Since as it is the build file is getting edited.

Reply:
this is redundant now